api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -714,6 +686,10 @@ private boolean userHasEverLoggedIn(
     user.setLastModifiedTime(now);
 
     updateInstitutionalAffiliations(updatedProfile, user);
+    if (workbenchConfigProvider.get().featureFlags.requireInstitutionalVerification) {
+      updatedProfile.setVerifiedInstitutionalAffiliation(
```

Comment:
AFAICT, calling `setVerifiedInstitutionalAffiliation` on the updatedProfile object has no effect, since this object is not persisted or returned in the HTTP response. Is it necessary?

Reply:
yep, i totally read this wrong before. argh